### ISSUE TITLE
White strip at the bottom of the page when Mist starts

### ISSUE THREAD

**Issue by chevdor** [ethereum/mist#1259]
#### System information

```
Version: `0.8.4`
OS & Version: osx
Node type: `parity/geth`
```

The issue is not new to 0.8.4.

See the white strip below:
![screen_shot_2016-10-05_at_15_44_54s](https://cloud.githubusercontent.com/assets/738724/19115801/9001741e-8b13-11e6-9ebc-ee983722c1f1.png)

After resizing the window a tiny bit, it goes away:
![screen_shot_2016-10-05_at_15_45_00s](https://cloud.githubusercontent.com/assets/738724/19115800/90015876-8b13-11e6-8cc6-50f4639a34b3.png)

**Comment by luclu** [comment]
pinging @alexvandesande

**Comment by chevdor** [comment]
This is **not** part of the HTML of the wallet. It is part of Mist itself.

**Comment by evertonfraga** [comment]
This is due to an old Electron issue (https://github.com/electron/electron/issues/5110), that was fixed on #1015 and came back, apparently.

**Comment by chevdor** [comment]
In 0.8.6, still there.

**Comment by evertonfraga** [comment]
PR available at #1457.

**Comment by evertonfraga** [comment]
Fixed in the released 0.8.8
